docker/quickstart/docker-compose-m1.quickstart.yml

```diff
@@ -146,7 +146,7 @@ services:
     - ELASTICSEARCH_INDEX_BUILDER_MAPPINGS_REINDEX=true
     - ELASTICSEARCH_INDEX_BUILDER_SETTINGS_REINDEX=true
     - ELASTICSEARCH_BUILD_INDICES_CLONE_INDICES=false
-    - GRAPH_SERVICE_IMPL=elasticsearch
+    - GRAPH_SERVICE_IMPL=neo4j
```

Comment:
Lets not change the graph service impl here. If you'd like to make it configurable, it can be like ```GRAPH_SERVICE_IMPL=${GRAPH_SERVICE_IMPL:-elasticsearch}``` so that you can override the impl with an environment variable if that's your preferred quickstart backend.

Reply:
done. Please check